src/arch/xtensa/lib/cpu.c

```diff
@@ -46,7 +47,7 @@ static void alloc_shared_secondary_cores_objects(void)
 {
 	uint8_t *dynamic_vectors;
 
-	dynamic_vectors = rzalloc(SOF_MEM_ZONE_RUNTIME_SHARED, 0, 0, SOF_DYNAMIC_VECTORS_SIZE);
+	dynamic_vectors = rballoc(SOF_MEM_FLAG_UNCACHE, 0, SOF_DYNAMIC_VECTORS_SIZE);
```

Comment:
It's better in English to say ```SOF_MEM_COHERENT``` here as it makes the usage more obvious.

Reply:
> It's better in English to say `SOF_MEM_COHERENT` here as it makes the usage more obvious.

makes sense, let me change.